api/restapi.py

```diff
@@ -84,7 +84,7 @@ def parse_profile(profile):
                     "profile": {
                         "name": profile.name,
                         "location": str(CountryCode.Name(profile.location)),
-                        "enryption_key": profile.encryption_key.encode("hex"),
```

Comment:
it might be a pain in the butt, but with a language like python, defining these keys as constants will go a long way in avoiding runtime errors due to typos in years to come.